style.css

```diff
@@ -1,14 +1,102 @@
-.title_3 {
-  color: rgb(0, 0, 0);
-  text-align: center;
+.page-portfolio {
+  background-image: url(images/bg-circle.png);
+  background-repeat: repeat;
+  background-attachment: fixed;
 }
 
-.section_info,
-.section_footer {
-  background: linear-gradient(90deg, #e7ecfb, #f3dbee);
+.page-portfolio-header,
+.page-portfolio-header-second {
+  font-family: monospace;
+  font-size: 22px;
+  color: #1e444f;
+  font-weight: 500;
+  letter-spacing: 2px;
+  text-align: justify;
+  margin: 10px;
+  padding: 10px;
+  line-height: 1.5;
+}
+
+.page-portfolio-header-name {
+  text-decoration: underline;
+  font-weight: bold;
+  text-align: justify;
+}
+
+.page-portfolio-header-image {
+  border-radius: 50%;
+  border: 3px solid #fff;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  float: right;
+  margin: 0 50px 75px;
+  width: 210px;
+}
+
+.page-portfolio-header-info,
+.section_footer_text,
+.page-portfolio-table-header,
+.page-portfolio-table,
+.section_footer_links a,
+.page-portfolio-info {
+  font-family: Arial, monospace, sans-serif;
+  font-size: 16px;
+  color: #284b63;
+  font-weight: 400;
+  line-height: 1.5;
+  text-align: justify;
+  margin: 10px;
+  padding: 10px;
+  text-indent: 15px;
+}
+
+.page-portfolio-header-laptop {
+  box-sizing: border-box;
+  border: 3px solid #fff;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  width: 40%;
+  margin: 25px 50px 75px;
+}
+
+.page-portfolio-header-second {
+  text-align: center;
+  font-weight: bold;
 }
 
-.white-bg-text-centered,
 .section_info_skills {
-  background-color: white;
+  font-family: monospace;
```

Comment:
Usually it is not recommended to use multiple fonts in a webpage. What you normally do is just set it once on the body and everything inside the body will inherit the font family from it.

Reply:
Fixed